front/src/components/atoms/DropdownButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
is `fortawesome` a typo? 😂

Reply:
No it is not! But I agree that their name is misleading...